CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
+# 3.1.5
+
+Fixes incorrect dropdown placement in IE11 #1733
+
 # 3.1.4
 
-Fixes incorrect dropdown placement in IE.
+Fixes incorrect dropdown placement in Edge #1733
```

Comment:
Not sure we should change release notes for already released versions. Maybe just highlight the fact the previous release only fixed the issue in Edge in the current versions release notes?

Also can you link the issue number to the issue please?